timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -346,7 +346,7 @@ export class TimeGraphChart extends TimeGraphChartLayer {
                 opacity: 0.2,
                 position: {
                     x: mouseStartX,
-                    y: 0
+                    y: this.rowController.verticalOffset
```

Comment:
If you mouse-wheel up/down while right-click zooming, you can see a gap on top of original vertical offset.

I believe the proper fix is to change height to :

Math.max(this.stateController.canvasDisplayHeight, this.rowController.totalHeight)

Reply:
Updated. Thanks.